### PR TITLE
Add GHA to build images

### LINKED ISSUE #27: CI
We should build the Docker image in CI and push it to GHCR. An image should be available for `linux/arm64`.

### PR DESCRIPTION
**Pull request by amberstarlight**
Fixes #27

This adds a GitHub Actions workflow to build images for `linux/arm64` and `linux/amd64`.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,10 @@
-FROM node:16.14-alpine
-COPY . /usr/cutiebot
+FROM node:lts-alpine
+LABEL org.opencontainers.image.source https://github.com/cutie-club/cutiebot
```
Comment (mini-ninja-64) [inline Dockerfile:2]: Good idea :100:

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,10 @@
-FROM node:16.14-alpine
-COPY . /usr/cutiebot
+FROM node:lts-alpine
```
Comment (mini-ninja-64) [inline Dockerfile]: would setting v16 explicitly be better, as when v18 becomes lts (2022-10-25) our app may stop building successfully

**Review by mini-ninja-64 — CHANGES_REQUESTED** [review]
Not really a change request just a thought

File: Dockerfile (hunk 1 above)
Comment (amberstarlight) [reply Dockerfile:2]: That's how you associate the image with this repo 😄

**Review by mini-ninja-64 — APPROVED** [review]
LGTM

**Pull request merged**